File: src/Web/WebAngular/src/app/domain/view-models/pipeline-log.view-model.ts
export interface PipelineLogViewModel {
	id: string;
	pipelineId: string;
	exitCode: number;
	runN: number;
	stdout: string | null;
	instructionWithErrorId: string | null;
	instructionWithError: string;
	triggeredBy: string;
	startTime: string;
	duration: string;
}

File: src/Core/Houston.Application/ViewModel/PipelineLogViewModels/PipelineLogViewModel.cs
namespace Houston.Application.ViewModel.PipelineLogViewModels {
	public class PipelineLogViewModel {
		public Guid Id { get; set; }

		public Guid PipelineId { get; set; }

		public int ExitCode { get; set; }

		public long RunN { get; set; }

		public string? Stdout { get; set; }

		public Guid? InstructionWithErrorId { get; set; }

		public string InstructionWithError { get; set; } = null!;

		public Guid? TriggeredById { get; set; }

		public string TriggeredBy { get; set; } = null!;

		public DateTime StartTime { get; set; }

		public TimeSpan Duration { get; set; }
	}
}
